cluster/gce/configure-vm.sh

```diff
@@ -386,7 +378,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: '$(echo "$MINION_IP_RANGE" | sed -e "s/'/''/g")'
+  cbr-cidr: 10.123.45.0/30
```

Comment:
Any thoughts on this? I'm giving cbr0 a garbage range (same on all nodes) at startup. This was easier than letting docker initially start up with docker0 and then adjusting docker's init args.

I think this would let docker schedule 1 pod (10.123.45.2) before cbr0 actually gets set up after kubelet runs. This will blackhole traffic from this node to that range for a brief amount of time at startup, so if some pre-kubelet step happened to need to talk to 10.123.45.{0-3}, this would break it.

Once kubelet runs in a pod, do we need it to have an actually routable IP assigned by docker, or is it just going to use the host IP?

I'd love to hear suggestions on a better way to do this if anybody has ideas.